site/content/entry/yuzu-linux-installer/index.md

```diff
@@ -0,0 +1,55 @@
++++
+date = "2022-08-10T23:00:00+05:30"
+title = "New Feature Release - Installer for Linux"
+author = "CaptV0rt3x"
+forum = 612296
++++
+
+Hey there yuz-ers!
+Guess what? We've got some news to share.
+We're excited to announce Linux support for our yuzu installer!
+Let's jump right in!
+
+<!--more-->
+
+## What's happening?
+
+We are very excited to announce Linux compatibility for our yuzu installer.
+The installer will distribute AppImages of mainline and early access builds for our Linux users.
+
+You can now download this installer from our [downloads page](https://yuzu-emu.org/downloads/).
+
+(Note: You need to be a yuzu Patreon Subscriber to get access to the early access builds)
+
+## Why?
+
+When we originally released our installer, our Patreon userbase was predominantly Windows users and truth-be-told, it still is.
+That's why the installer's Linux compatibility wasn't a priority for us then.
+
+{{< imgs
+	"./maintenance.png| As a current limitation, Patreon users will have to paste their token by pressing Ctrl + V"
+  >}}
+
+But, over the years we've seen consistent growth in the number of our Linux users 
+and realised that we had to ensure that they reaped the benefits of the installer too.
+
+{{< imgs
+	"./install.png| It's never a bad idea to pick both builds"
+  >}}
+
+Thanks to the efforts of [toastUnlimited](https://github.com/lat9nq), who implemented Linux compatibility for the installer,
+we're now able to address this long due feature request.
+
+{{< imgs
+	"./download.png| Remember we update daily!"
+  >}}
+
+## Fin
+
+As with any new update, we've internally tested this to some extent. 
+However, it is very easy for us to have missed something or for you to encounter a wierd bug or issue.
+So, please reach out to us on our [Discord](https://discord.gg/u77vRWY) and report these bugs, so that we can quickly address them.
```

Comment:
```suggestion
We could have easily missed an edge case or some weird bug or issue.
So, please reach out to us on our [Discord](https://discord.gg/u77vRWY) and report any bugs you find, so that we can quickly address them.
```